Makefile

```diff
@@ -1,13 +1,23 @@
+RUBY_VERSION := "$(shell ruby -v)"
+RUBY_VERSION_REQUIRED := "$(shell cat .ruby-version)"
+RUBY_MATCH := $(shell [[ "$(shell ruby -v)" =~ "ruby $(shell cat .ruby-version)" ]] && echo matched)
```

Comment:
@slonka I tried using `ifneq` but I couldn't make it work.
something like the following should do the trick, but `findstring` isn't finding the string, any ideas?
`ifneq (, $(findstring $(RUBY_VERSION_REQUIRED),$(RUBY_VERSION)))`

We use `=~` because the ruby version might contain info about the patch and system architecture, e.g.
`ruby 3.1.2 (2022-12-25 revision a528908271) [arm64-darwin22]`